db/migrate/20151213185304_add_slug_to_page.rb

```diff
@@ -0,0 +1,6 @@
+class AddSlugToPage < ActiveRecord::Migration
+  def change
+    add_column :pages, :slug, :string, null: false
```

Comment:
When the database already contains pages, that error is raised:
```
== 20151213185304 AddSlugToPage: migrating ====================================
-- add_column(:pages, :slug, :string, {:null=>false})
rake aborted!
StandardError: An error has occurred, this and all later migrations canceled:

PG::NotNullViolation: ERROR:  column "slug" contains null values
```
It's tricky because we can't put a default value...